spec/requests/api/v1/allocations_controller_index_spec.rb

```diff
@@ -69,7 +69,25 @@
         end
       end
 
+      describe 'filtering results by status' do
+        let(:unfilled_allocation) { create :allocation, :unfilled }
+        let(:filled_allocation) { create :allocation, :filled }
+        let(:cancelled_allocation) { create :allocation, :cancelled }
+        let!(:allocations) { [unfilled_allocation, filled_allocation, cancelled_allocation] }
+        let(:params) { { filter: { status: 'cancelled' } } }
+
+        it 'filters the results' do
+          expect(response_json['data'].size).to be 1
+        end
+
+        it 'returns the allocation that matches the filter' do
```

Comment:
can you add a test for a list of statuses?

Reply:
I think that's suitably covered off with the finder specs - this is really just to prove that the request params make it through the controller to the finder and would probably be overkill to test everything again. The test did fail until I remembered to whitelist the extra `status` param in the controller :)